src/freertos_drivers/esp32/Esp32HardwareTwai.cxx

```diff
@@ -876,9 +876,11 @@ void* twai_watchdog(void* param)
         {
             LOG(INFO,
                 "ESP-TWAI: "
-                "RX:%d (pending:%zu,overrun:%d,discard:%d,missed:%d,lost:%d) "
-                "TX:%d (pending:%zu,suc:%d,fail:%d) "
-                "Bus (arb-loss:%d,err:%d,state:%s)",
+                "RX:%" PRIu32 " (pending:%zu,overrun:%" PRIu32
+                ",discard:%" PRIu32 ",missed:%" PRIu32 ",lost:%" PRIu32 ") "
+                "TX:%" PRIu32 " (pending:%zu,suc:%" PRIu32
+                ",fail:%" PRIu32 ") "
+                "Bus (arb-err:%" PRIu32 ",err:%" PRIu32 ",state:%s)",
```

Comment:
https://github.com/atanisoft/OpenMRNIDF/pull/7/files#diff-88ee088b2bbc83933050861edc298188dbc20d3dbc31bf2fa003d9ae7c989603 has changed all of the types for these values from uint32_t to size_t which cleans up this log statement considerably.

Reply:
Let's do that in a different round. I have not reviewed the 5.1.0 changes yet.

Reply:
That's fine, the bulk of the changes you have already pulled in, it's mostly further cleanup and adjustments for 5.1/5.2 and those are minimilar IIRC